test/lib/specHelperSpec.js

```diff
@@ -88,25 +94,45 @@ describe('Fluxxed up test helpers', function() {
 
 
     it('has a value in the page', () => {
-      expect(rig.domNode.find('.answer').text()).to.match(/initial value$/)
+      expect(rig.domNode.find('.answer').text()).to.match(/initial value/)
     })
 
     it('updates the form', (done) => {
       // Manipulate the DOM:
-      rig.fillIn('input', 'new thing')
-      rig.clickLink('Submit')
+      rig.fillIn('input[name="text-input"]', 'new thing')
+
       rig.toggleCheckbox('#checkbox-input')
+      rig.setValue('.select-input', '1')
 
+      expect(rig.domNode.find('#checkbox-input')[0].checked).to.match(/true/)
```

Comment:
looks like you are checking that the element actually changes. This is a valid thing to test, but it should be in its own test.